scripts/create_all_packages.sh

```diff
@@ -80,6 +80,11 @@ if [ "${DRYRUN}zz" == "zz" ]; then
     # Change permissions on source files to match those expected by the server
     chmod 644 build/*
     CURRENT_DATE=$(date +'%Y-%m-%d')
+    # Rename files to include the date and remove extraneous info so that permalinks can be generated
+    for file in build/*; do
+        target=$(sed -E "s/-[0-9.]+commit[^.]+/_$CURRENT_DATE/" <<<"$file")
```

Comment:
When we create a tag (for a released version), we go through the same code. We have a different file name, that will not match this regexp.
It will generate an error with `mv` because `$file` and `$target` would be the same file.
It should be simple to just not to do the `mv` in this case.

Reply:
OK, thanks. Good idea.